app/result.py

```diff
@@ -0,0 +1,8 @@
+from app.knight import Knight
+
+
+class Result:
+    @classmethod
+    def return_result(cls, knights: list[Knight]):
+        result = {knight.name: knight.hp for knight in knights}
+        return result

```

Comment:
make it as a function, if you want but there is no need to use class and classmethod